### PR TITLE
Afsaneh/add mt5 account

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
this PR cover adding MT5 BVI accounts to Fiat wallet accounts, it requires login with fully authenticated account.
CU: https://app.clickup.com/t/20696747/WALL-3292

### CODE REVIEW

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
The standard at the moment is to leave out ; 
image

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

File: cypress/support/e2e.js [hunk 1]
```diff
@@ -108,11 +108,14 @@ Cypress.Commands.add('c_doOAuthLogin', (app) => {
   cy.c_visitResponsive(Cypress.env("oAuthUrl"),"large")
   //To let the dtrader page load completely
   cy.get('.cq-symbol-select-btn', { timeout: 10000}).should('exist')
-  cy.findByTestId('launch-modal').then(($element) =>{
-    if($element){
+  cy.wait(3000)
```
Comment (markodonnell-deriv) [inline cypress/support/e2e.js:111]: Is this necessary?

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Do we ned the wait?

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**